v2/thirdparty/custom-ui/thirdparty-login.mdx

```diff
@@ -323,9 +327,9 @@ Coming Soon
 
 <TabItem value="ios">
 
-<h3>Step 1) Fetching the authorisation token on the frontend</h3>
+<h4>Step 1) Fetching the authorisation token on the frontend</h4>
 
```

Comment:
instead of linking to exact lines, instead, say find the `loginWithApple` function in this file